src/components/NewAppModal.js

```diff
@@ -147,12 +159,21 @@ export const NewAppModal = _.flow(
   const contents = Utils.switchCase(viewMode,
     ['createWarn', renderCreateWarning],
     ['deleteWarn', renderDeleteWarning],
+    ['launchWarn', renderLaunchWarning],
     [Utils.DEFAULT, renderDefaultCase]
   )
 
+  const WarningTitle = ({ children }) => {
```

Comment:
Per our discussion - this is copied from ClusterManager so probably both could be extracted into a common component in cluster-common